LobbyServer2/LobbyServer/LobbyServerProtocol.cs

```diff
@@ -645,6 +675,40 @@ public void HandleCheckAccountStatusRequest(CheckAccountStatusRequest request)
                 ResponseId = request.RequestId
             };
             Send(response);
+            
+            if (LobbyConfiguration.IsTrustWarEnabled())
+            {
+                PersistedAccountData account = DB.Get().AccountDao.GetAccount(AccountId);
+
+                int omni = GetTotalXPByFactionID(account, 0);
+                int evos = GetTotalXPByFactionID(account, 1);
+                int warbotics = GetTotalXPByFactionID(account, 2);
+
+                Send(new PlayerFactionContributionChangeNotification()
+                {
+                    CompetitionId = 1,
+                    FactionId = 0,
+                    AmountChanged = 0,
+                    TotalXP = omni,
+                    AccountID = account.AccountId,
+                });
+                Send(new PlayerFactionContributionChangeNotification()
+                {
+                    CompetitionId = 1,
+                    FactionId = 1,
+                    AmountChanged = 0,
+                    TotalXP = evos,
+                    AccountID = account.AccountId,
+                });
+                Send(new PlayerFactionContributionChangeNotification()
+                {
+                    CompetitionId = 1,
+                    FactionId = 2,
+                    AmountChanged = 0,
+                    TotalXP = warbotics,
+                    AccountID = account.AccountId,
+                });
```

Comment:
Is it supposed to be a loop?

Reply:
there is no otherway that i saw to send all 3 at once